### DISCUSSION TITLE
Communication between a host page and a JupyterLite instance running in an IFrame: Showing No Kernel Found

### DISCUSSION THREAD

**Discussion by gautamsarawagi** [jupyterlite/jupyterlite#1340] [Q&A] [answered]
I am building an application where I want to interact with the host page and also automatically executes the code in the first load. 

For that I came across this [document](https://jupyterlite.readthedocs.io/en/stable/howto/configure/advanced/iframe.html).

It is working completely fine but during the execution it is showing this: 

![image](https://github.com/jupyterlite/jupyterlite/assets/101802666/ee152b79-34d8-4e86-894e-daa6b046050b)

It would be a great help if I can know how to setup the kernels also.

**Comment by jtpio** [comment]
> It would be a great help if I can know how to setup the kernels also.

There is some documentation about this here: https://jupyterlite.readthedocs.io/en/stable/howto/configure/kernels.html

**Reply by gautamsarawagi** [reply, marked as answer]
Thanks a lot @jtpio it worked.
